src/xo-app/index.js

```diff
@@ -149,16 +149,12 @@ export default class XoApp extends Component {
                   <h4>{context.name}</h4>
                 </Col>
               </Row>,
-              ...map(context.shortcuts, ({ message, keys }) => message &&
-                <Row>
+              ...map(context.shortcuts, ({ message, keys }, key) => message &&
+                <Row key={key}>
```

Comment:
Context should have a key too and this key should be namespaced by the key context.